From 6986a536b0cf9956987e2cb8e2b955420e18d195 Mon Sep 17 00:00:00 2001
From: Chih-Wei Huang <cwhuang@linux.org.tw>
Date: Tue, 26 Feb 2019 10:56:45 +0800
Subject: [PATCH 6/9] android: ignore unimportant compile warnnings

Signed-off-by: Chih-Wei Huang <cwhuang@linux.org.tw>
---
 src/Android.mk | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/src/Android.mk b/src/Android.mk
index 008d5b2..2f85a3a 100644
--- a/src/Android.mk
+++ b/src/Android.mk
@@ -60,7 +60,11 @@ $(GEN): $(LOCAL_PATH)/intel_version.h.in $(wildcard $(LOCAL_PATH)/../.git/logs/H
 	sed -e "s|\@INTEL_DRIVER_GIT_VERSION\@|$$VER|" $< > $@
 LOCAL_GENERATED_SOURCES += $(GEN)
 
-LOCAL_CFLAGS := -DLINUX -g -Wall -Wno-unused -fvisibility=hidden
+LOCAL_CFLAGS := -DLINUX -g -Wall -Wno-unused -fvisibility=hidden \
+	-Wno-missing-field-initializers \
+	-Wno-unused-parameter \
+	-Wno-pointer-arith \
+	-Wno-sign-compare \
 
 LOCAL_SHARED_LIBRARIES := libdl libdrm libdrm_intel libcutils \
                libva libva-android libstdc++
@@ -71,5 +75,3 @@ LOCAL_SHARED_LIBRARIES += liblog
 endif
 
 include $(BUILD_SHARED_LIBRARY)
-
-
-- 
2.20.1

